mobile/android/build.gradle

```diff
@@ -34,14 +34,15 @@ buildscript {
         minSdkVersion = 21
         targetSdkVersion = 28
         compileSdkVersion = 28
+        buildToolsVersion = "28.0.3"
     }
     repositories {
         google()
         jcenter()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.3'
-        classpath 'com.google.gms:google-services:4.0.2'
+        classpath 'com.google.gms:google-services:+'
```

Comment:
Is this okay?  I remember all these java libs requiring very specific versions for compat between eachother.

Reply:
I'm not sure if anything changed recently, but the version that we had previously didn't work. So I had to change this.